shared_model/backend/protobuf/common_objects/signature.hpp

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
Maybe rename file to proto_signature?

Reply:
Imo we should remove `proto_` prefix, it's in the protobuf directory already. Or at least introduce some nice logic rules for that